tests/template_tests/test_extends.py

```diff
@@ -117,3 +119,11 @@ def test_unique_history_per_loader(self):
         template = engine.get_template('base.html')
         output = template.render(Context({}))
         self.assertEqual(output.strip(), 'loader2 loader1')
+
+    @override_settings(INSTALLED_APPS=[
+        'template_tests.extended',
+        'template_tests.base',
+    ])
+    def test_get_extended_base_template_w_extended_included_template(self):
+        template = get_template("base_site.html")
```

Comment:
I strongly agree that it would be a lot more transparent to have it inline but unfortunately it would no longer lean on the behavior that I am aiming to fix in the app_directory loader in that case.  I did fix the single quotes (sorry about that).  I am open to other approaches that will improve transparency but I was unable to come up with anything reasonable that still tugs on the app_directory loader and I did not see any other tests that managed to evade using fake apps for that purposes.  Please let me know if there is anything else I can do.